### PR TITLE
test: Add test case for soul-spring-boot-starter-plugin.

### PR DESCRIPTION
**Pull request by HoldDie**
Add test case for soul-spring-boot-starter-plugin.

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by HoldDie** [comment]
image

down. @yu199195

File: soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-springcloud/src/test/java/org/dromara/soul/springboot/starter/client/springcloud/SoulSpringCloudClientConfigurationTest.java [hunk 1]
```diff
@@ -53,10 +64,16 @@
     @Autowired
     private SoulSpringCloudConfig soulSpringCloudConfig;
 
+    @BeforeClass
+    public static void before() throws Exception {
+        PowerMockito.mockStatic(RegisterUtils.class);
+        PowerMockito.doNothing().when(RegisterUtils.class, "doRegister", Mockito.any(), Mockito.any(), Mockito.any());
+    }
```
Comment (dengliming) [inline soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-springcloud/src/test/java/org/dromara/soul/springboot/starter/client/springcloud/SoulSpringCloudClientConfigurationTest.java:71]: Do we have to import `powermock`? `Mockito ` also supports mock static methods. Here's a example [https://github.com/dromara/soul/blob/master/soul-client/soul-client-common/src/test/java/org/dromara/soul/client/common/utils/RegisterUtilsTest.java](https://github.com/dromara/soul/blob/master/soul-client/soul-client-common/src/test/java/org/dromara/soul/client/common/utils/RegisterUtilsTest.java) Can you try it?

Comment (HoldDie) [reply soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-springcloud/src/test/java/org/dromara/soul/springboot/starter/client/springcloud/SoulSpringCloudClientConfigurationTest.java]: Modify the code by using Mockito

```java
MockedStatic<RegisterUtils> registerUtilsMockedStatic = mockStatic(RegisterUtils.class);
registerUtilsMockedStatic.when(() -> RegisterUtils.doRegister(any(), any(), any())).thenAnswer((Answer<Void>) invocation -> null);
```

Comment (HoldDie) [reply soul-spring-boot-starter/soul-spring-boot-starter-client/soul-spring-boot-starter-client-springcloud/src/test/java/org/dromara/soul/springboot/starter/client/springcloud/SoulSpringCloudClientConfigurationTest.java]: image

**Review by dengliming — APPROVED** [review]
Thanks. We'll merge it when the CI passes.

**Pull request merged**